src/components/PictureCard.js

```diff
@@ -27,9 +26,9 @@ export default function PictureCard(props) {
 
   return (
     <>
-      <Grid container style={{ padding: '20px', maxWidth: '1200px', margin: 'auto', backgroundColor: "#F2F2F2" }}>
+      <Grid container style={{ padding:'20px 0 20px 0', maxWidth: '1200px', margin: 'auto', backgroundColor: "#F2F2F2" }}>
```

Comment:
Can remove the last 2 values from padding
```suggestion
      <Grid container style={{ padding:'20px 0', maxWidth: '1200px', margin: 'auto', backgroundColor: "#F2F2F2" }}>
```